search/includes/classes/class-queue.php

```diff
@@ -53,7 +55,7 @@ public function is_enabled() {
 	}
 
 	public function setup_hooks() {
-		add_action( 'edit_terms', [ $this, 'offload_indexing_to_queue' ] );
+		add_action( 'edited_terms', [ $this, 'offload_term_indexing_to_queue' ], 0 );
 		add_action( 'pre_delete_term', [ $this, 'offload_indexing_to_queue' ] );
```

Comment:
```suggestion
		add_action( 'pre_delete_term', [ $this, 'offload_indexing_to_queue' ], 0, 2 );
```
(zero just for consistency with the `edited_term`, unless default `10` is important

Reply:
Can't do that with deletes since offloading to cron creates a race condition between WordPress deleting the stuff and whatever delay there is until the cron job runs.

Reply:
Ah, I see